packages/apollo-server-integration-testsuite/src/ApolloServer.ts

```diff
@@ -2695,15 +2695,23 @@ export function testApolloServer<AS extends ApolloServerBase>(
         });
       });
 
-      it('unsubscribes from schema update on close', async () => {
-        const unsubscribeSpy = jest.fn();
-        const { gateway, triggers } = makeGatewayMock({ unsubscribeSpy });
-        triggers.resolveLoad({ schema, executor: async () => ({}) });
-        await createApolloServer({ gateway });
-        expect(unsubscribeSpy).not.toHaveBeenCalled();
-        await stopServer();
-        expect(unsubscribeSpy).toHaveBeenCalled();
-      });
+      // Serverless frameworks execute ApolloServer.start() in a dangling
```

Comment:
Another alternative is to perform an action that requires startup to have finished, eg `await server.executeOperation({query: '{__typename}'})`

Reply:
That sounds fine to me, I've re-enabled the test for serverless and added the above code to ensure it's started.